src/core/sayt.ts

```diff
@@ -98,6 +98,42 @@ export interface ProductSearchConfig {
   productSort?: any;
 }
 
+export interface AutocompleteResponse {
+  status: {
+    code: number;
+    internalCode: number;
+    message: 'string';
+    additionalInfo: any;
```

Comment:
We should probably keep the same structure as the additionalInfo below
```suggestion
    additionalInfo: {
      [key: string]: any;
    };
```

Reply:
The one below was explicitly set to that in the API files, however, this one wasn't and can be `null` instead.

Reply:
Ah cool 👍